resources/patches/ungoogled-chromium/fix-building-without-mdns-and-service-discovery.patch

```diff
@@ -1,73 +1,56 @@
-# Fix building with enable_service_discovery=false and enable_mds=false
```

Comment:
I think we can at least keep the first comment line of this patch.

Reply:
Done.